views/7confirm-new-job-role.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Add New Job Role</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+    </head>
+    <body>
+        <header class="bg-dark text-light">
+            <div class="container">
+                <div class="row">
+                <div class="col-md-12">
+                    <h1>Add New Job Role</h1>
+                </div>
+            </div>
+            </div>
+        </header>
+        <main class="container mt-4">
+            <section id="addNewJobTitleHeader" class="mt-4">
+                <h2>Please confirm the following information is correct</h2>
+            </section>
+            <section id="form" class="bg-light py-5">
+                <div class="container">
+                    {% if errormessage %}
+                    <div class="alert alert-danger" role="alert">
+                        {{ errormessage }}
+                    </div>
+                    {% endif %}
+                    <form action="/admin-confirm-new-job-role" method="post">
```

Comment:
There doesn't seem to be an "/admin-confirm-new-job-role" route in your controller. How are you handling this request?

Reply:
During the merge my controller got deleted... I've added it back now sorry about that